hash/nes.xml

```diff
@@ -39933,7 +40531,7 @@ license:CC0
 		<description>Trog! (Euro)</description>
 		<year>1991</year>
 		<publisher>Acclaim Entertainment</publisher>
-		<info name="serial" value="NES-4A-(NOE/ESP)"/>
+		<info name="serial" value="NES-4A-(AUS/NOE/ESP)"/>
```

Comment:
Same with this – it’s confusing and easy to misinterpret.  Just split it into separate tags.